### DISCUSSION TITLE
how to open my dialog when I click library open event

### DISCUSSION THREAD

**Discussion by zhangwei900808** [excalidraw/excalidraw#8405] [General]
<img width="306" alt="image" src="https://github.com/user-attachments/assets/9670868c-ffc7-49b2-bcbb-fb55a8276828">

**Comment by zhangwei900808** [comment]
when I click the open action I hope that set open my dialog

**Comment by Mrazator** [comment]
AFAIK this is not configurable ATM, you would have to fork.
